hydpy/models/lland/lland_control.py

```diff
@@ -840,35 +842,73 @@ class KapGrenz(parametertools.Parameter):
     >>> wmax(100.0, 150.0, 200.0)
     >>> kapgrenz(option="0_WMax/10")
     >>> kapgrenz
-    kapgrenz([[0.0, 10.0],
-              [0.0, 15.0],
-              [0.0, 20.0]])
+    kapgrenz(option="0_WMax/10")
+    >>> kapgrenz.values
+    array([[ 0., 10.],
+           [ 0., 15.],
+           [ 0., 20.]])
 
     The third possible string is `FK/2_FK` where the lower and the upper
     threshold are 50 % and 100 % of the value of parameter |FK|, which does
     not correspond to any available `LARSIM`_ option:
 
     >>> kapgrenz(option="FK/2_FK")
     >>> kapgrenz
+    kapgrenz(option="FK/2_FK")
+    >>> kapgrenz.values
+    array([[ 30.,  60.],
+           [ 60., 120.],
+           [ 90., 180.]])
+
+    >>> kapgrenz
+    kapgrenz(option="FK/2_FK")
+
+    >>> kapgrenz.values = [[30.0, 60.0], [60.0, 120.0], [90.0, 180.0]]
+    >>> kapgrenz
+    kapgrenz(option="FK/2_FK")
+
+    If we change the values and they do not comply with the option, the values
+    instead of the option will be returned
+
+    >>> kapgrenz.values = 100.0
+    >>> kapgrenz
+    kapgrenz(100.0)
+
+    If we change the parameter |FK|, on which |KapGrenz| depends, |KapGrenz| will not
+    be updated, but will remain at its old values. The option is no longer returned
+    because it is not valid anymore.
+    >>> kapgrenz(option="FK/2_FK")
+    >>> kapgrenz
+    kapgrenz(option="FK/2_FK")
+    >>> kapgrenz.values
+    array([[ 30.,  60.],
+           [ 60., 120.],
+           [ 90., 180.]])
+    >>> fk(100)
+    >>> kapgrenz
     kapgrenz([[30.0, 60.0],
               [60.0, 120.0],
               [90.0, 180.0]])
 
-    Wrong keyword arguments result in errors like the following:
+    The option will also not be returned if it leads to nan values:
```

Comment:
returned -> printed ("?")
nan -> `nan` (or, better, |numpy.nan|)

blank line

Reply:
I corrected to printed and [numpy.nan|